src/Foundation/Enum.cs

```diff
@@ -1380,6 +1385,9 @@ public enum NSFileProtectionType {
 		CompleteUnlessOpen,
 		[Field ("NSFileProtectionCompleteUntilFirstUserAuthentication")]
 		CompleteUntilFirstUserAuthentication,
+		[Watch (10, 0), TV (17, 0), NoMac, iOS (17, 0), MacCatalyst (17,0)]
+		[Field ("NSFileProtectionCompleteWhenUserInactive")]
+		CompleteWhenUserInactive,
```

Comment:
This, is a normal situation, should not be allowed. YET! this enum was never released and was added in https://github.com/xamarin/xamarin-macios/pull/18918 meaning no developer ever has used this enum.